packages/common-common/src/tbcStatsDSender.ts

```diff
@@ -0,0 +1,57 @@
+import { StatsDController, StatsDTag } from 'common-common/src/statsd';
+import { BalanceProvider } from 'token-balance-cache/src';
+
+// Class with Helper functions to send statsD
+export class TbcStatsDSender {
```

Comment:
Since this is TBC-specific, we should probably put it in the TBC package, rather than in `common-common`, unless there's a way to make it reusable somehow.